Pipfile.lock

```diff
@@ -40,7 +40,7 @@
         "eq-translations": {
             "editable": true,
             "git": "https://github.com/ONSDigital/eq-translations.git",
-            "ref": "88cc5756ddf6166653f2c5695370f6727facd62c"
+            "ref": "e854cd4904caab72824fd33d9bd21ba0807f0160"
```

Comment:
I needed to change this to `5ffb3ec12a5f5fe67a3a2daef2e1fb366f14a47e` to test the new `__version__` functionality.

Reply:
Changed it, should pass GitHub Actions check now, thanks!👍